modules/vectorfieldvisualization/datastructures/integrallineset.cpp

```diff
@@ -55,25 +57,30 @@ inviwo::IntegralLine& IntegralLineSet::at(size_t idx) { return lines_.at(idx); }
 
 const inviwo::IntegralLine& IntegralLineSet::at(size_t idx) const { return lines_.at(idx); }
 
-void IntegralLineSet::push_back(IntegralLine &line) {
-    line.setIndex(lines_.size());
-    lines_.push_back(line);
+void IntegralLineSet::push_back(const IntegralLine& line, SetIndex updateIndex) {
```

Comment:
Yes, to make the code more readable. 
From the Google C++ Style Guide ([Implementation Comments § Function Argument Comments](https://google.github.io/styleguide/cppguide.html#Implementation_Comments))
 > Consider changing the function signature to replace a bool argument with an enum argument. This will make the argument values self-describing.